v2/cli/term/event_page.go

```diff
@@ -20,7 +20,10 @@ type eventPage struct {
 	eventInfo  *tview.TextView
 	workerInfo *tview.TextView
 	jobsTable  *tview.Table
+	eventLogs  *tview.TextView
+	logModal   tview.Primitive
 	usage      *tview.TextView
+	logsClient core.LogsClient
```

Comment:
I wouldn't add a new field for this. We already have reliable access to this client via `e.apiClient.Events().Logs()`.